corehq/apps/app_manager/models/common.py

```diff
@@ -2186,6 +2143,8 @@ def uses_usercase(self):
 
 
 class AdvancedForm(IndexedFormBase, NavMenuItemMediaMixin):
+    from corehq.apps.app_manager.models.schedules import FormSchedule
```

Comment:
this still gets run upon first read (classes creation isn't deferred the way function body execution is), so I'd be pretty surprised if these couldn't live up top